### ISSUE TITLE
Help training my dataset

### ISSUE THREAD

**Issue by gitdeepheolp** [clovaai/deep-text-recognition-benchmark#338]
So, i started creating my learning images using trdg , moved data.mdb lock.mdb to /data_lmdb_release/training/ and data_lmdb_release/validation 

used the command to train   : 
CUDA_VISIBLE_DEVICES=0 python3.8 train.py --train_data data_lmdb_release/training --valid_data data_lmdb_release/validation --select_data MJ-ST --batch_ratio 0.5-0.5 --Transformation TPS --FeatureExtraction ResNet --SequenceModeling BiLSTM --Prediction Attn --rgb --character '''!"#$%&\(\)\*+,-./0123456789:;=?-@ABCDEFGHIJKLMNOPQRSTUVWXYZ\_abcdefghijklmnopqrstuvwxyzАБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЫЬЭЮЯабвгдежзийклмно прстуфхцчшщыьэюя''' --sensitive  --output_channel 512 --saved_model saved_models/old/iter_200000.pth --imgH 50 --imgW 130 --adam

edited the chars on demo.py , but when i test it i get the following result  :

model input parameters 32 100 20 1 512 256 152 25 TPS ResNet BiLSTM Attn
loading pretrained model from best_accuracy.pth
/home/ex/.local/lib/python3.8/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
--------------------------------------------------------------------------------
image_path               	predicted_labels         	confidence score
--------------------------------------------------------------------------------
images/test/0.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/1.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/1.png        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/2.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/3.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/3.png        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/4.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/5.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/6.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/6.png        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/7.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/7.png        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/8.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/9.jpg        	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000
images/test/10.jpg       	@ЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1ИЩ1ИЩЩ1	0.0000

even with best_accuracy.pth i get no result.

demo command : python3.8 demo.py --Transformation TPS --FeatureExtraction ResNet --SequenceModeling BiLSTM --Prediction Attn --image_folder images/ --saved_model best_accuracy.pth --imgW 130 --imgH 50 --sensitive --rgb --output_channel 512 --num_fiducial 20

used on the same images that i created .mdb files. what i'm doing wrong ?

**Comment by slent310** [comment]
hi~The seam question. do you solve it?

**Comment by gitdeepheolp** [comment]
No, i'm still trying figure out how to add non-latin characters to my trained model.
